.docker/php/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:8.1.5-fpm
+FROM php:8.3.1-fpm
```

Comment:
docker should use the lowest version we support not the highest, that make sure that we code with the proper compatiblity.